src/pages/ProductList/index.jsx

```diff
@@ -0,0 +1,232 @@
+import Spiner from './components/Loader';
+
+
+const Container = styled.div`
+  display: flex;
+  margin-right: 8%;
+  .container__sidebar {
+    width: ${(props) => props.collapsed ? '4.25rem' : '36%'};
+    background-color: white;
+    overflow-y: hidden;
+  }
+  .container__content {
+    background-color: white;
+    padding: 1rem;
+    margin-left: 1rem;
+    width: -webkit-fill-available;
+  }
+`;
+
+const Sidebar = styled.section`
+  .sidebar__section {
+    display: flex;
+    align-items: center;
+  }
+  .sidebar__icon {
+    min-width: 4rem;
+    width: 4rem;
+    height: 5rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    img {
+      width: 2.5rem;
+      height: 2.5rem;
+    }
+  }
+  .check {
+    height: 3rem;
+  }
+  .sidebar__bullet {
+    width: 0.6rem;
+    height: 0.6rem;
+    background-color: black;
+    border-radius: 50%;
+  }
+`;
+
+const Pagination = styled.section`
+  nav {
+    margin: 3rem 0 2rem 0;
+    display: flex;
+    justify-content: right;
+  }
+  a {
+    min-width: 2rem;
+    border: 0.2rem solid ${Colors['BG-C']};
+    cursor: pointer;
+    font-size: 1.8rem;
+    padding: 0.5rem 1.5rem;
+    text-decoration: none;
+    color: black;
+    border-radius: 0.5rem;
+    &:hover {
+      background-color: brown;
+      color: white;
+    }
+  }
+`;
+
+const NoProducts = styled.div`
+  height: 8rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+
+const ProductList = () => {
+  const [isMenuCollapse,setIsMenuCollapse] = useState(false);
+  const [activeFilters, setActiveFilters] = useState([]);
+  const [categoryList, setCategoryList] = useState([]);
+  const [products, setProducts] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setCategoryList([...CategoryList['results']]);
+    setProducts([...productList['results']]);
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 2000);
+  }, []);
+
+  useEffect(() => {
+    const categoryNames = [];
+    activeFilters.forEach(element => {
+      categoryNames.push(element.data.name.toLowerCase());
+    });
+
+    if (categoryNames.length > 0) {
+      // eslint-disable-next-line max-len
+      const newListProducts = productList['results'].filter( item => categoryNames.includes(item.data.category.slug));
+      setProducts(newListProducts);
+    } else {
+      setProducts([...productList['results']]);
+    }
+  }, [activeFilters]);
+
+  const handleSidebar = () => setIsMenuCollapse(!isMenuCollapse);
+  
+  const handlerCheckbox = (e,id) => {
+    const isActive = e.target.checked;
+    if (isActive) {
+      const filterSelected = categoryList.filter(item => item.id === id);
+      setActiveFilters(activeFilters.concat(filterSelected));
+    } else {
+      const newActiveFilters = activeFilters.filter(item => item.id !== id);
+      setActiveFilters(newActiveFilters);
+    }
+  }
+  
+  return (
+    <Container collapsed={isMenuCollapse}>
```

Comment:
When you collapse the sidebar, the filters names cant be properly shown, so its difficult to know what filters are active:
![image](https://user-images.githubusercontent.com/26983421/158598941-fee8f8d3-b34d-4c6a-ac85-04f87b8fe28a.png)
And the footer could be stick to bottom, take a look at this https://css-tricks.com/couple-takes-sticky-footer/